ext/xmlrpc/xmlrpc.stub.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+function xmlrpc_encode($value): string {}
+
+function xmlrpc_decode(string $xml, string $encoding = "iso-8859-1") {}
+
+function xmlrpc_decode_request(string $xml, &$method, string $encoding = "iso-8859-1") {}
```

Comment:
Same as above:
```suggestion
function xmlrpc_decode_request(string $xml, &$method, string $encoding = "iso-8859-1"): void {}
```

Reply:
Test seems to disagree: https://github.com/php/php-src/blob/e3abc534eb98add83d9f705d864f1d0fdd79e5ea/ext/xmlrpc/tests/002.phpt#L12-L14

Reply:
Indeed. Same as above.